### PR TITLE
feat: Add brand colour names & replace darts with point

### PR DESCRIPTION
**Pull request by katjam**
I tried to preserve contrast & maintain branding.

Not sure if we should delete the unused stuff or keep here for ref for pairs and colouring.

https://github.com/wearedarts/games/pull/10

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
   <body>
     <header class="page-section">
       <div class="container">
-        <img alt="darts" src="./images/darts-logo-cream.svg" class="logo" />
+        <img alt="darts" src="./images/the-point-logo-cream.svg" class="logo" />
```
Comment (Nikomus) [inline src/index.html:12]:
```suggestion
        <img alt="the point." src="./images/the-point-logo-cream.svg" class="logo" />
```

**Comment by Nikomus** [comment]
Should the footer link stay as darts, or change to the point?

**Comment by katjam** [comment]
Not sure... maybe both? Let's make a point to ask them what they want in the footer - after alpha.

**Comment by katjam** [comment]
Is this OK to go in?

**Review by Nikomus — APPROVED** [review]
Yep, let's go for this 🏡

**Pull request merged**